modules/deploy/pages/redpanda/manual/resilience/shadowing.adoc

```diff
@@ -0,0 +1,677 @@
+
+== What gets replicated
+
+Shadowing replicates your topic data with complete fidelity, preserving all message records with their original offsets, timestamps, headers, and metadata. The partition structure remains identical between source and shadow clusters, ensuring applications can resume processing from the exact same position after failover.
+
+Consumer group data flows according to your group filters, replicating offsets and membership information for matched groups. ACLs replicate based on your security filters. Schema Registry data synchronizes schema definitions, versions, and compatibility settings.
+
+The following topic properties are always replicated:
+
+* Partition count
+* `max.message.bytes`
+* `cleanup.policy`
+* `message.timestamp.type`
+
+The following topic properties are never replicated:
+
+* `redpanda.remote.readreplica`
+* `redpanda.remote.recovery`
+* `redpanda.remote.allowgaps`
+* `redpanda.virtual.cluster.id`
+* `redpanda.leaders.preference`
+* `redpanda.cloud_topic.enabled`
+
+
+By default, Redpanda always syncs the following topic properties:
+
+* `compression.type`
+* `retention.bytes`
+* `retention.ms`
+* `delete.retention.ms`
+* `replication.factor`
+* `min.compaction.lag.ms`
+* `max.compaction.lag.ms`
+
+Additional topic properties are replicated only when you explicitly specify them in your `synced_shadow_topic_properties` configuration. Set `exclude_default` to `true` if you want to sync only the properties listed in `synced_shadow_topic_properties`. The filtering system you configure determines the precise scope of replication across all components, allowing you to balance comprehensive disaster recovery with operational efficiency.
+
+=== Set filters
+
+Filters determine which resources Shadowing automatically creates when establishing your shadow link.
+
+Topic filters select which topics Shadowing automatically creates as shadow topics when they appear on the source cluster. Once Shadowing creates a shadow topic, it continues replicating until you fail over the topic, delete it, or delete the entire shadow link.
+
+Consumer group and ACL filters control which groups and security policies replicate to maintain application functionality.
+
+==== Filter types and patterns
+
+Each filter uses two key settings:
+
+* **Pattern type**: Determines how names are matched
+  - `LITERAL`: Matches names exactly (including the special wildcard `*` to match all items)
+  - `PREFIX`: Matches names that start with the specified string
+* **Filter type**: Specifies whether to include or exclude matching items
+  - `INCLUDE`: Replicate items that match the pattern
+  - `EXCLUDE`: Skip items that match the pattern
+
+==== Filter processing rules
+
+Redpanda processes filters in the order you define them with exclude filters taking precedence. Design your filter lists carefully:
```

Comment:
Below you use INCLUDE and EXCLUDE, ^^ and here you use lowercase. The context seems more or less the same, so I think they should all be written the same. Whatever you decide to go with, just be consistent.